doc/authentication.md

```diff
@@ -122,6 +128,8 @@ For instructions on how to set up Candid-based authentication, see the [Candid a
 
 (authentication-rbac)=
 ## Role Based Access Control (RBAC)
+```{youtube} https://www.youtube.com/watch?v=VE60AbJHT6E
```

Comment:
the other videos have an empty line before the link, not sure if this makes any difference though

Reply:
It doesn't, but always better to be consistent. Fixed. :)